### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by arpit01923**
Avatar component is added to docs

### CODE REVIEW

File: documentation/docs.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                    style="width: 880px; height: 322px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
```
Comment (hrishikeshmane) [inline documentation/docs.html]: You can maintain indentation, either manually or by prettier extension in VScode

File: documentation/docs.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+                    <li>Medium</li>
+                    <li>Large</li>
+                    <li>Extra-large</li>
+                    <li>Extra Extra-large</li>
+                </ol>
+                <br>
+                <h2>Example of Avatar</h2>
+                <br>
+                <p class="font-1_5x">-Round Avatar</p>
+                <br><br>
```
Comment (hrishikeshmane) [inline documentation/docs.html]: you can use self closing tag as well- <br />

File: documentation/docs.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (hrishikeshmane) [inline documentation/docs.html]: Maybe change title to - Web Beauty | Document

File: documentation/docs.html (hunk 1 above)
Comment (arpit01923) [reply documentation/docs.html]: Done

File: documentation/docs.html (hunk 2 above)
Comment (arpit01923) [reply documentation/docs.html]: done

File: documentation/docs.html (hunk 3 above)
Comment (arpit01923) [reply documentation/docs.html]: Changed

**Comment by manojncsoni007** [comment]
Looks good

**Review by manojncsoni007 — COMMENTED** [review]
looks good

**Pull request merged**